Dockerfile

```diff
@@ -1 +1 @@
-docker/cycamore-ci/Dockerfile
\ No newline at end of file
+docker/master-ci/Dockerfile
```

Comment:
This seems to by a symlink to the wrong file - should be the file for building develop?